website/content/sdk/installation.md

```diff
@@ -4,37 +4,39 @@ sidebar_position: 1
 description: The OpenBB SDK provides programmatic access to all Terminal functions. This layer of code allows users to build their own tools and applications on top of the existing architecture. Follow these steps to install on a local machine.
 keywords: [installation, installer, install, guide, mac, windows, linux, python, github, macos, how to, explanation, openbb, sdk, api, pip, pypi,]
 ---
-The OpenBB SDK provides programmatic access to all Terminal functions and much more. This layer of code allows users to build their own tools and applications on top of the existing architecture. Follow these steps to install on a local machine.
+The OpenBB SDK provides programmatic access to all Terminal functions and much more. This layer of code allows users to build their own tools and applications on top of the existing architecture.
 
 :::note
 If the OpenBB Terminal has already been installed from source code, no additional installation steps are required. **You already have the SDK installed.**
+Verify the installation by following [these steps](#verify-installation).
 :::
 
-## Minimal Installation
-
-For users who have basic developer tools installed and are using Python through a virtual environment, we recommend the following minimal installation method:
+## Requirements
 
-```shell
-pip install openbb
-```
-
-This method provides access to the data aggregation and charting functions of the OpenBB SDK. It does not provide access to the advanced features that are provided by the Portfolio Optimization and Machine Learning toolkits.
+The OpenBB SDK is designed to be used in a virtual environment. __You should not try installing the SDK into your system python or your base conda enviromnent__. If you are not sure what this means, please follow the instructions to install OpenBB Terminal from source [(link)](/terminal/installation/source).
 
-If need to use the SDK with the Machine Learning and Portfolio Optimization toolkits we recommend following the [Full Installation instructions](#full-installation).
+OpenBB SDK is available on [PyPI](https://pypi.org/project/openbb/). To install the SDK from PYPI you will need to meet the following requirements:
 
-## Full Installation
+- Use Python 3.8, 3.9, or 3.10
+- Use Python from a virtual environment
+- Have a C compiler installed (e.g. `gcc`, `clang`, etc.). These usually come with the operating system.
+- Have `cmake` installed on your machine. This is usually available through your package manager (e.g. `apt`, `brew`, `yum`, etc.).
```

Comment:
why not following the docs to install the terminal from source - it has cmake bundled with the conda environment

Reply:
So basically if the Terminal one is a complete version and this constantly references that one, why does this version even exist? Either it needs to include the same depth as Terminal or it might as well not exist imo. Otherwise you are required to constantly ping-pong between the two and people will definitely struggle with it which increases the load on Darren within Discord.

When we have 4.0 it will be as simple as a `pip install` but in the meantime it should just be complete documentation.

Reply:
> . There are people who prefer to use vanilla pip and venv over other things. For people like these we acknowledge that there is a package on pypi and the nightly builds.

This is exactly why we need to explicitly state `--no-cache-dir` because people will install into whatever container they already have, using the version of SciPy Numpy, etc. that are in cache. 

There is a notable difference in that installing from PyPi does not accomplish the same thing as a Git Clone, and people will end up trying to `poetry install` their `pip install openbb` installation.